Gemfile

```diff
@@ -39,8 +41,8 @@ end
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
+  # gem 'spring'
```

Comment:
Any reason why we're commenting these out?

Reply:
`rails generate` and `rails test` won't work because `spring` is already running. I have to stop spring each time I try to run tests. This helps to fix the problem.

Reply:
Cool. Resolving